### DISCUSSION TITLE
The dreaded 404.

### DISCUSSION THREAD

**Discussion by PKDrinker** [community/community#106874] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

# 404 There isn't a Github page site here.
This is my repo [https://github.com/OmniaAi-solutions/OmniaAi-solutions.github.io](url). I have been trying to deploy this site but I am not able do that. I have read a lot of previously asked questions and checked everything but everything seems to be okay. I don't know why the site is not getting deployed.
Can anyone please help me?

**Comment by seyLu** [comment]
#### Hi @PKDrinker,

Have you tried reading the [documentation for GitHub Pages](https://docs.github.com/en/pages/quickstart#creating-your-website), on how to set it up?

![image](https://github.com/community/community/assets/98249191/25282d62-7d15-47e8-9210-948931c44a80)

**Reply by PKDrinker** [reply]
Yes. And I have already deployed a website from another account and that works fine.

**Comment by ZephyrusBlaze** [comment]
Can you link the repository?

**Reply by PKDrinker** [reply]
https://github.com/OmniaAi-solutions/OmniaAi-solutions.github.io
